week.go

```diff
@@ -86,11 +86,11 @@ func (w *Week) Add(weeks int) (Week, error) {
 	return New(year, week)
 }
 
-// Sub calculates the positive difference between w and u (u-w) in number of weeks
+// Sub calculates the positive difference between w and u (w-u) in number of weeks
 func (w *Week) Sub(u *Week) int {
```

Comment:
Conceptually `Week` represents a value type and therefore methods and functions should receive week values as non pointers. Making `u` a non pointer would make method more consistent with other methods.